### DISCUSSION TITLE
switching image on  the /icub/camcalib output port

### DISCUSSION THREAD

**Discussion by patrickshas** [robotology/community#617] [Q&A] [answered]
Hello,

We are using a Python script to take in /icub/cam/left and /icub/cam/ right which works fine (displays on display port fine, etc.)
When we switch our video input feed to /icub/camcalib/left/out and /icub/camcalib/right/out the display port video start switching between calibrated and uncalibrated videos (almost like a choppy video edit).

Any ideas what might this be caused by?
Thanks in advance

**Comment by paliasgh** [comment]
Hello,

Attaching a screenshot here:

![Screenshot from 2023-05-02 16-57-11](https://user-images.githubusercontent.com/59978262/235784993-b4c48d3b-03e1-4ce4-9d78-221eb450753b.png)

These two yarpviews show the calibrated videos (`/icub/camcalib/left/out` and `/icub/camcalib/left/out`) until we start Python script. Then, `/icub/camcalib/left/out` and `/icub/camcalib/left/out` become like the screenshots. As you can see, the upper part of the left image is calibrated and the rest is uncalibrated. The entire right image is uncalibrated.

**Comment by pattacini** [comment, marked as answer]
Hi,

You may consider sharing the Python script (maybe through a [Gist](https://gist.github.com/)) and any other piece of material that could help us narrow down the analysis.
At any rate, from what you reported, it looks like two streams are displaying images on the same `yarpview`.

You may thus try the following to check how many ports are connected to the display one:
```console
yarp ping /icub/view/left
```

Removing an unwanted connection can be achieved via the command `yarp disconnect /port-1 /port-2`.
